guides/doc-Release_notes/topics/foreman.adoc

```diff
@@ -4,8 +4,63 @@
+The new Host Details page is no longer experimental, although it is still not the default one. One can change `Settings` -> `New hosts details UI` to Yes to make it the default option. There are few new tabs added on this page,
+
+==== Insights tab
+The recommendations are available for the hosts in the Insights tab when using RHEL with a Red Hat cloud subscription.
+It is feasible to begin remediation for the suggested remedies (if any are available).
+
+==== Ansible tab
+It has the ability to manage Ansible roles and variables.
+The recurring jobs for the host can also be scheduled from the Ansible tab.
+
+==== PatternFly 4 Search tab
+A contemporary search experience is provided by the new search bar. It's the beginning of an eventual transition to PatternFly 4 for many UI components in the future.
+
+==== New Build button and card for BMC Power Management
+New modals for a build button and BMC Power Management is available, which improves exprience of new Host Details page.
+
+=== Descriptions for all of the provisioning templates
+Over a hundred new descriptions have been added to provisioning templates, snippets, and scripts to assist users understand particular usage and use them in an effective manner.
+
+=== New purge:puppet rake task
+If you don't intend to use the Puppet ENC functionality then this rake task can be used to purge Environments, Puppetclasses, Smartclass parameters and Config groups.
+
+=== Cron job to clean old Audits
+A new cron job has been introduced to purge the old Audits on a daily basis. The new `Saved audits interval` setting allows you to specify the number of days you want to save Audits for.
+
+=== Provisioning snippets support Puppet 7
+The provisioning snippets have been updated to setup Puppet 7 repositories, as well as configure and install packages from those repositories.
+
+=== Performance improvements for index pages and Host Config Status
+The new caching approach speeds up index pages loading while consuming less memory.
+The Host Config Status is now more efficient, using fewer requests and less memory.
+
 [id="foreman-upgrade-warnings"]
 == Upgrade Warnings
 
+=== Running Foreman on Ubuntu 18.04 dropped
+
+Foreman 2.5 introduced support to run on Ubuntu 20.04. Foreman 3.0 deprecated support for running on Ubuntu 18.04 and Foreman 3.1 drops it.
+Users are encouraged to upgrade to Ubuntu 20.04 either on Foreman 2.5 or 3.0.
+
+Note that this is only support to run Foreman and Foreman Proxy themselves on Ubuntu 18.04. Managing Ubuntu 18.04 systems remains supported. See https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008[the RFC] for more information.
+
 [id="foreman-deprecations"]
 == Deprecations
+
+=== The `:unattended` Setting has been deprecated
+
+Historically Foreman has supported the `:unattended` setting in `settings.yaml`. This is barely tested and has seen various regressions because of that.
```

Comment:
I added this sentence compared to the ones on the website. While writing https://github.com/theforeman/theforeman.org/pull/1939 I changed it slightly. I'm not sure what the best phrasing is, but we should align them all :)